src/lib/cosmos.decorator.tsx

```diff
@@ -0,0 +1,215 @@
+import { Web3Provider } from '@ethersproject/providers'
+import { UnsupportedChainIdError, useWeb3React, Web3ReactProvider } from '@web3-react/core'
+import {
+  InjectedConnector,
+  NoEthereumProviderError,
+  UserRejectedRequestError as UserRejectedRequestErrorInjected,
+} from '@web3-react/injected-connector'
+import { NetworkConnector } from '@web3-react/network-connector'
+import { useEffect, useState } from 'react'
+
+const RPC_URLS: { [chainId: number]: string } = {
```

Comment:
Use the constants already defined in the repo.
We're going to need to move them to `lib` eventually, but for now I think we should use them as-is.